openscap_daemon/dbus_daemon.py

```diff
@@ -434,7 +449,12 @@ def containers(self):
         """Used by `atomic scan`. Do not break this interface!
         """
         import docker
-        docker_conn = docker.Client()
+        # Class docker.Client was renamed to docker.APIClient in
+        # python-docker-py 2.0.0.
+        try:
+            docker_conn = docker.APIClient()
+        except AttributeError:
+            docker_conn = docker.Client()
```

Comment:
you repeat this code around a lot, what if instead you assigned the ctor to a variable at the top of the module and reused that variable? that way you only have to document it once

Reply:
actually I guess it's fine, I wish it didn't repeat as much code but I can live with it :)